src/store/refreshIllustrations.js

```diff
@@ -33,11 +34,11 @@ export default async function refresh() {
 
     return acc;
   }, {});
-  await updateDict('entities', updates);
+  await updateDict('illustrations', updates);
 
   // Mark missing illustrations as such
   const archiveIllustrationIds = archive.map(({ id }) => id);
-  const localEntityEntries = await ENTITIES.entries();
+  const localEntityEntries = await ILLUSTRATIONS.entries();
   const localEntitiyIds = localEntityEntries.map(([id]) => id);
```

Comment:
Do we want to change the variable name here and on Line 41 to Illustrations instead of Entities as well?